plugin/manager.go

```diff
@@ -173,9 +173,22 @@ func (pm *Manager) reload() error { // todo: restore
 		if validFullID.MatchString(v.Name()) {
 			p, err := pm.loadPlugin(v.Name())
 			if err != nil {
-				return err
+				if os.IsNotExist(errors.Cause(err)) {
+					// Likely some error while removing on an older version of docker
+					logrus.WithError(err).WithField("id", filepath.Base(v.Name())).Warn("Missing plugin config, skipping. This may be caused due to a failed remove and requires manual cleanup.")
+				} else {
+					logrus.WithError(err).WithField("id", filepath.Base(v.Name())).Error("Error loading plugin, skipping.")
+				}
+				continue
 			}
 			plugins[p.GetID()] = p
+		} else {
+			if validFullID.MatchString(strings.TrimSuffix(v.Name(), "-removing")) {
+				// There was likely some error while removing this plugin, let's try to remove again here
+				if err := system.EnsureRemoveAll(v.Name()); err != nil {
+					logrus.WithError(err).WithField("id", filepath.Base(v.Name())).Warn("error while attempting to clean up previously removed plugin")
```

Comment:
Looks like the other messages are all lowercase, so I suggest changing the ones above to be lowercase as well